pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Links
+
+- [Jira task]()
+- [Upstream repository]()
+- [Changelog/diff]()
+- [Related PR(s)]()
```

Comment:
Upstream dependency listings? This would save reviewers' time, so they don't have to dig through upstream repo to find them